.gitignore

```diff
@@ -23,3 +23,5 @@ retworkx/*pyd
 **/*.so
 retworkx-core/Cargo.lock
 **/.DS_Store
+venv/
+requirements.txt
```

Comment:
This can be an issue in the future so I will remove, if we ever add `requirements.txt`
```suggestion
```